backend/templates/requirements.mustache

```diff
@@ -7,5 +7,6 @@ Flask-PyMongo
 swagger-ui-bundle >= 0.0.9
 python-dotenv
 pyjwt[crypto]
-explainaboard == 0.8.4
+explainaboard >= 0.8.8
```

Comment:
I think it may be easier to maintain if we use a specific version number for the web app because we usually won't be able to keep up with the updates of the SDK. If a breaking change is introduced in the SDK and this repo is behind, it also blocks us from redeploying the web app safely. For example, I scaled down the dev environment server two weeks ago. At that time, a newer version of the SDK has been released with breaking changes. If the requirements file uses `>= 0.8.4`, it will automatically upgrade to the latest SDK which would break the deployment.